book_review/app/views/book_ratings/index.html.slim

```diff
@@ -0,0 +1,23 @@
+h1 Listing book_ratings
+
+table
+  thead
+    tr
+      th Rating value
+      th Book
+      th
+      th
+      th
+
+  tbody
+    - @book_ratings.each do |book_rating|
```

Comment:
you can place content for each book rating to file _book_rating.html.slim and render it simply with command

= render @book_ratings